roachtest: activerecord failed

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4407748&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4407748&tab=artifacts#/activerecord) on master @ [91dfbc8e740ba5792834534dd41af6fb85cee721](https://github.com/cockroachdb/cockroach/commits/91dfbc8e740ba5792834534dd41af6fb85cee721):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:190,orm_helpers.go:116,activerecord.go:251,test_runner.go:779: 
		Tests run on Cockroach v22.1.0-alpha.1-1433-g91dfbc8e74
		Tests run against activerecord 6.1
		6641 Total Tests Run
		6640 tests passed
		1 test failed
		19 tests skipped
		2 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: ConcurrentTransactionTest#test_transaction_isolation__read_committed (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_1) is available in the artifacts' activerecord log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #69080 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-21.1]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4427196&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4427196&tab=artifacts#/activerecord) on master @ [4c65a10fea9762b97559c1a488cd5b96cb2cf81a](https://github.com/cockroachdb/cockroach/commits/4c65a10fea9762b97559c1a488cd5b96cb2cf81a):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:190,orm_helpers.go:116,activerecord.go:251,test_runner.go:779: 
		Tests run on Cockroach v22.1.0-alpha.1-1511-g4c65a10fea
		Tests run against activerecord 6.1
		6641 Total Tests Run
		6640 tests passed
		1 test failed
		19 tests skipped
		2 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: CockroachDB::PostgresqlIntervalTest#test_interval_type (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_1) is available in the artifacts' activerecord log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #69080 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4455314&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4455314&tab=artifacts#/activerecord) on master @ [978744dbb94b03497154547ce7fa97ea3be9a401](https://github.com/cockroachdb/cockroach/commits/978744dbb94b03497154547ce7fa97ea3be9a401):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:190,orm_helpers.go:116,activerecord.go:251,test_runner.go:779: 
		Tests run on Cockroach v22.1.0-alpha.1-1656-g978744dbb9
		Tests run against activerecord 6.1
		6641 Total Tests Run
		6640 tests passed
		1 test failed
		19 tests skipped
		2 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: ConcurrentTransactionTest#test_transaction_isolation__read_committed (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_1) is available in the artifacts' activerecord log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #69080 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/cmd/roachtest/tests/activerecord_blocklist.go
+++ b/pkg/cmd/roachtest/tests/activerecord_blocklist.go
@@ -36,13 +36,19 @@ var activeRecordBlockList21_1 = blocklist{}
 
 var activeRecordBlockList20_2 = blocklist{}
 
-var activeRecordIgnoreList22_1 = activeRecordIgnoreList21_2
+var activeRecordIgnoreList22_1 = blocklist{
+	"ConcurrentTransactionTest#test_transaction_isolation__read_committed": "flaky - https://github.com/cockroachdb/activerecord-cockroachdb-adapter/issues/237",
+	"FixturesTest#test_create_fixtures":                                    "flaky - FK constraint violated sometimes when loading all fixture data",
+	"IgnoreFixturesTest#test_ignores_books_fixtures":                       "flaky - FK constraint violated sometimes when loading all fixture data",
+	"IgnoreFixturesTest#test_ignores_parrots_fixtures":                     "flaky - FK constraint violated sometimes when loading all fixture data",
+}
 
 var activeRecordIgnoreList21_2 = blocklist{
 	"ActiveRecord::CockroachDB::UnloggedTablesTest#test_gracefully_handles_temporary_tables": "modified to pass on 20.2",
-	"FixturesTest#test_create_fixtures":                "flaky - FK constraint violated sometimes when loading all fixture data",
-	"IgnoreFixturesTest#test_ignores_books_fixtures":   "flaky - FK constraint violated sometimes when loading all fixture data",
-	"IgnoreFixturesTest#test_ignores_parrots_fixtures": "flaky - FK constraint violated sometimes when loading all fixture data",
+	"ConcurrentTransactionTest#test_transaction_isolation__read_committed":                   "flaky - https://github.com/cockroachdb/activerecord-cockroachdb-adapter/issues/237",
+	"FixturesTest#test_create_fixtures":                                                      "flaky - FK constraint violated sometimes when loading all fixture data",
+	"IgnoreFixturesTest#test_ignores_books_fixtures":                                         "flaky - FK constraint violated sometimes when loading all fixture data",
+	"IgnoreFixturesTest#test_ignores_parrots_fixtures":                                       "flaky - FK constraint violated sometimes when loading all fixture data",
 }
 
 var activeRecordIgnoreList21_1 = blocklist{

